elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -40,6 +40,7 @@ export interface P2PPenalty {
 export interface P2PPeerInfo {
 	readonly ipAddress: string;
 	readonly wsPort: number;
+	readonly secret?: number;
```

Comment:
We should not put the secret inside the P2PPeerInfo or else it will be shared with other peers and they can use it to guess our buckets.